README.md

```diff
@@ -15,9 +15,9 @@ worldengine world -s 1 -n seed1
 
 Worlds are generated using plate simulations, erosion, rain shadows, Holdridge life zones model and plenty of other phenomenons.
 
-Once a world it can be used for simulation civs evolution (see project [civs](https://github.com/ftomassetti/civs)).
+A finished world can be used for simulating the evolution of a civilization (see project [civs](https://github.com/ftomassetti/civs)).
```

Comment:
Should I use "The created world..." or "A stored world..." instead? I think I will go with the former one for now.